src/test/java/com/baidu/hugegraph/functional/HugeClientHttpsTest.java

```diff
@@ -65,6 +66,7 @@ public void testHttpsClientBuilderWithConnection() {
     @Test
     public void testHttpsClientWithConnectionPoolNoUserParam() {
         client = HugeClient.builder(BASE_URL, GRAPH)
+                           .configUser(USERNAME, PASSWORD)
```

Comment:
maybe should not add username for test case testHttpsClientWithConnectionPoolNoUserParam